js/js.js

```diff
@@ -27,6 +26,5 @@ module.exports = {
 	last: require('./last/last'),
 	makeArray: require('./make-array/make-array'),
 	setImmediate: require('./set-immediate/set-immediate'),
-	string: require('./string/string'),
-	types: require('./types/types')
```

Comment:
Ok, I can do this differently. The only reason this was done was because the main `can-util.js` file extends`can-namespace` with this object and `can-types` and `can-cid` now take care of adding themselves to the namespace.